e621_dl.py

```diff
@@ -140,7 +140,7 @@ def search_pools(
         ignore_pagination=True,
     )
     for pool in pools:
-        directory = download_dir / pool.name
+        directory = download_dir / re.sub(r'[<>:\"/\|?*]', '', pool.name)
```

Comment:
We should compile all the regexes beforehand. Similar to how VALID_FILE_NAME is done.

Additionally, I'll need to test this regex a little bit on my own time. Hopefully, we will be able to merge it today-tomorrow.